api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -513,14 +532,18 @@ type HostedClusterSpec struct {
 
 	// ImageContentSources specifies image mirrors that can be used by cluster
 	// nodes to pull content.
-	//
+	// When imageContentSources is set, the controllers will generate a machineConfig.
+	// This MachineConfig will be part of every payload generated by the controllers for any NodePool of the HostedCluster.
+	// Changing this value will trigger a rollout for all existing NodePools in the cluster.
```

Comment:
Just a thought, should we add some moniker for fields like this that trigger rollouts so anyone could easily search for such fields.

Like `// +rollout` or `// +causesRollout` or something else.